crates/sui-types/src/bridge.rs

```diff
@@ -42,6 +42,15 @@ pub const BRIDGE_SUPPORTED_ASSET: &[&str] = &["btc", "eth", "usdc", "usdt"];
 pub const BRIDGE_COMMITTEE_MINIMAL_VOTING_POWER: u64 = 6000; // out of 10000 (100%)
 pub const BRIDGE_COMMITTEE_MAXIMAL_VOTING_POWER: u64 = 10000; // (100%)
 
+// Threshold for action to be approved by the committee (our of 10000)
+pub const APPROVAL_THRESHOLD_TOKEN_TRANSFER: u64 = 3334;
+pub const APPROVAL_THRESHOLD_EMERGENCY_PAUSE: u64 = 450;
```

Comment:
Where do I find out more about the thresholds, and does this allow very easy pausing of the bridge?